impl/container/src/main/java/io/serverlessworkflow/impl/container/executors/CommandPropertySetter.java

```diff
@@ -15,20 +15,38 @@
  */
 package io.serverlessworkflow.impl.container.executors;
 
+import static io.serverlessworkflow.impl.WorkflowUtils.isValid;
+
 import com.github.dockerjava.api.command.CreateContainerCmd;
 import io.serverlessworkflow.api.types.Container;
-import java.util.function.Function;
+import io.serverlessworkflow.impl.TaskContext;
+import io.serverlessworkflow.impl.WorkflowContext;
+import io.serverlessworkflow.impl.WorkflowDefinition;
+import io.serverlessworkflow.impl.WorkflowModel;
+import io.serverlessworkflow.impl.WorkflowUtils;
+import io.serverlessworkflow.impl.WorkflowValueResolver;
+import java.util.Optional;
+
+class CommandPropertySetter implements ContainerPropertySetter {
 
-class CommandPropertySetter extends ContainerPropertySetter {
+  private Optional<WorkflowValueResolver<String>> command;
 
-  CommandPropertySetter(CreateContainerCmd createContainerCmd, Container configuration) {
-    super(createContainerCmd, configuration);
+  CommandPropertySetter(WorkflowDefinition definition, Container configuration) {
+    String commandName = configuration.getCommand();
```

Comment:
During parsing time (when the constructor is invoked) we create the filter to resolve the command